### PR TITLE
small wording changes

### PR DESCRIPTION
**Pull request by swrhim**
Something I'm not sure what the wording should be is stores -> reactors : trigger. I'm not sure how to make this explicit. Thoughts?

### CODE REVIEW

File: diagrams/context.puml [hunk 1]
```diff
@@ -73,12 +73,12 @@ services   <-R-> stores     : Query/Transact\n[in terms of events]
 stores       ..> publishers : are followed by
 stores       ..> reactors   : trigger
 
-reactors     --> services   : feed back into
+reactors     --> services   : feed data back into
```
Comment (bartelink) [inline diagrams/context.puml:76]: I'm intentionally being vague/general here - a reaction is driving a decision and/or could be triggering idempotent command processing driven by a process manager. I'm not sure what a better term for that might be, but _data_ seems overly specific and makes it sound more like an Ingester (which probably can benefit from being cast as doing something straightforward)

File: diagrams/context.puml [hunk 2]
```diff
@@ -73,12 +73,12 @@ services   <-R-> stores     : Query/Transact\n[in terms of events]
...
 reactors     ..> outputs    : generate complex\nnotifications for
 reactors    .U.> caches     : maintain views in
 
 publishers   ..> outputs    : emit complex notifications to
-publishers   ..> outputs    : render some events to
```
Comment (bartelink) [inline diagrams/context.puml:81]: here, I'm trying to convey that it's being selective in what it opts to render (whereas for complex notifications, it's publishing every one that's been prepared by the "Compose Notifications" umbrella thing (which is another reason why one might not be feeding just `data` above))

**Comment by bartelink** [comment]
>Something I'm not sure what the wording should be is stores -> reactors : trigger. I'm not sure how to make this explicit. Thoughts?

I updated the label to be *fresh writes trigger* since...

Can you review and/or rebase this based on recent updates?

File: diagrams/context.puml (hunk 1 above)
Comment (swrhim) [reply diagrams/context.puml:76]: Right now, I think it's too vague. Saying even feed decision back into the service would make this a lot clearer

File: diagrams/context.puml (hunk 2 above)
Comment (swrhim) [reply diagrams/context.puml]: how about, render filtered events to?

File: diagrams/context.puml (hunk 1 above)
Comment (bartelink) [reply diagrams/context.puml:76]: https://github.com/jet/equinox/blob/diag/DOCUMENTATION.md now says "manage follow-on reactions via"

File: diagrams/context.puml (hunk 2 above)
Comment (bartelink) [reply diagrams/context.puml]: image

**Comment by swrhim** [comment]
@bartelink If we can resolve the last comment, I'd want to abandon this PR and have you update the change directly on master. Lots of conflicts and doesn't seem worth it to resolve them for small changes

**Pull request closed without merging**